### DISCUSSION TITLE
Difference between "npx @melt-ui/cli init" and "npm install"?

### DISCUSSION THREAD

**Discussion by markfswe** [melt-ui/melt-ui#685] [Q&A] [answered]
`npx @melt-ui/cli@latest init` places `@melt-ui/melt-ui` to `devDependencies`, however in "manual install" instructions command installs `@melt-ui/melt-ui` to `dependencies`. I don't think this difference is intended. What is the correct way to install melt-ui?

![image](https://github.com/melt-ui/melt-ui/assets/49815452/8f684f20-e6ef-4b49-b495-2423e045cc69)

**Comment by TGlide** [comment, marked as answer]
Definitely not intended! In the end, with how bundling and Svelte works, it doesn't matter. Putting it in devdependencies is totally fine, and we should update docs to reflect this.

**Reply by markfswe** [reply]
Fine. Would like to see the docs updated. I can handle this. Or if you are going to fix this, could you please link this discussion?

**Comment by TGlide** [comment]
Docs were fixed :)
